proto/recipe_specification.proto

```diff
@@ -18,8 +19,11 @@ message RecipeSchema {
   // What resources this plugin can handle with their parameter constraints
   repeated ResourcePattern supported_resources = 4;
   
+  // What scheduling capabilities this plugin supports
+  SchedulingCapability scheduling = 5;
+  
   // Plugin requirements
-  PluginRequirements requirements = 5;
+  PluginRequirements requirements = 6;
```

Comment:
Okay for now, but in general you never renumber fields in a protobuf - it breaks backwards compatibility for any already encoded messages

Reply:
ok, will keep in mind